packages/ember-data/lib/system/store.js

```diff
@@ -440,9 +440,36 @@ Store = Ember.Object.extend({
     store.find('person', { page: 1 });
     ```
 
-    This will ask the adapter's `findQuery` method to find the records for
-    the query, and return a promise that will be resolved once the server
-    responds.
+    By passing an object {page: 1} as an argument to the find method, it
+    delegates to the adapter's findQuery method. The adapter then makes
+    a call to the server, transforming the object {page: 1} as parameters
+    that are sent along, and will return a RecordArray when the promise
+    resolves.
+
+    Exposing queries this way seems preferable to creating an abstract query
+    language for all server-side queries, and then require all adapters to
+    implement them.
+
+    The call made to the server, using a Rails backend, will look something like this:
+
+    ```
+    Started GET "/api/v1/person?page=1"
+    Processing by Api::V1::PersonsController#index as HTML
+    Parameters: {"page"=>"1"}
+    ```
+
+    If you do something like this:
+    ```javascript
```

Comment:
This needs a newline between the end of the "If you do something like this:" line and the start of the javascript code block otherwise the it doesn't seem to get rendered correctly.

![screen shot 2015-01-24 at 3 15 38 pm](https://cloud.githubusercontent.com/assets/54056/5888843/e3b18034-a3db-11e4-9427-acad986e0277.png)